data/tlg0018/tlg001/tlg0018.tlg001.1st1K-eng1.xml

```diff
@@ -0,0 +1,1013 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">The Works of Philo Judaeus</title>
+<sponsor>University of Leipzig</sponsor>
+<funder>European Social Fund Saxony</funder>
+<principal xml:id="GRC">Gregory Crane</principal>
+<respStmt>
+<persName xml:id="Jouve">Jouve</persName>
+<resp>OCR-ed, did a first pass of correction on, and encoded the text</resp>
+</respStmt>
+<respStmt>
+<persName>Gregory Crane</persName>
+<resp>In 2023, GRC broke the translations up into works, cleaned up tagging of notes, and used digital files from CCEL.org to follow the sections as the appear in the David Scholer edition of the Yonge translation. He corrected many errors in the text but many remain.</resp>
```

Comment:
this is not what goes in the resp tag: it goes in the change tag (see my comment below)--
important because the resp statement is what is in the Scaife sidebar and we use a few predetermined labels for consistency with editors

Reply:
As an example, the information here ("Digital conversion" and editing and "CTS conversion") are what is in `<resp>`
<img width="319" alt="image" src="https://github.com/OpenGreekAndLatin/First1KGreek/assets/4633998/763ffb47-f832-4c48-b707-f1ef325bcbe5">